pom.xml

```diff
@@ -4,6 +4,13 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.5.3</version>
```

Comment:
musisz uważać na wersję, polecałbym używać tych z kursu, żeby gładko przejść całość

W firmach nadal często spotkasz spring boota v2 więc to nie problem wielki ;)